libqtile/resources/default_config.py

```diff
@@ -25,6 +25,7 @@
 # SOFTWARE.
 
 from libqtile import bar, layout, qtile, widget
+from libqtile.backend.wayland.inputs import InputConfig
```

Comment:
This would fail if the Wayland dependencies are not present.

Reply:
You could include these lines in the default config but comment them out. You can also add some comments to explain what they're for.

@jwijenbergh may have a different idea.

Reply:
You can also do the import in an if backend is Wayland block. But at this point it's better to just make NumLock the default and have a false version commented?